policies/project-director-election-process.md

```diff
@@ -0,0 +1,60 @@
+# Project Director Election Process
+
+## Aims
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria, focusing on shared power and new leadership rather than solely prioritizing the best candidate.
+    * Strive for tolerance and inclusivity, aiming for the overlap of tolerance rather than the overlap of preference.
+* Conduct the selection process in a time-efficient manner.
+* Incorporate input from a wide range of stakeholders across the entire Rust project.
+* Ensure a diverse pool of qualified candidates for selection.
+* Create a psychologically safe environment that values minority perspectives and encourages positive feedback, encouragement, and tolerance rather than preference or criticism.
+* Ensure participants feel positive about their interactions and the execution of the process.
+
+## Description
+
+### Setup
+
+* The council selects a facilitator by consent:
+    * Any team member can suggest a facilitator or volunteer themselves.
+    * The facilitator must not be a nominee in the election and must be someone everyone trusts to set aside personal preferences during the election process.
+    * **Aims of the facilitator role**
+        * Clearly documenting and communicating the process and announcing and supporting its execution.
+        * Tracking, reviewing, and documenting the effectiveness of the process.
+* Review or Define Role Description ([example](https://github.com/rust-lang/rfcs/pull/3392#issuecomment-1505697944))
+    * Aims
+    * Time Investment
+    * Responsibilities
+    * Qualifications
+    * Term
+
+### Pre-Meeting Coordination
+
+* Gather nominations
+    * Council representatives collaborate with their subteams to collect nominations on behalf of their branch of the organization tree.
+    * Subteams can utilize any process they prefer, but we recommend following a similar approach to the process outlined in the "Election Meeting" section below
+* Share nominations publicly
+    * Each team representative must publicly share their team's nominations at least ten days before the final election meeting. This allows other teams to share feedback or objections about candidates their team didn't initially consider with their representative.
+    * This 10-day period is equivalent to the Change Round in the "Election Meeting" step; representatives may change their nominations in advance of the meeting
+
+### Election Meeting
+
+* Share nominations
```

Comment:
I realized, at least in the way I'm envisioning the process, we have a bit of ambiguity now behind what counts as a nomination.

In the Pre-meeting Coordination step, I imagined each team would nominate some number of candidates (preferably equal to the number of open Project Director seats, but the number could be smaller or larger, or even zero). Then we union all these candidates together to get a pool of candidates.

Then in the Election Meeting, the council members draw from the pool of candidates to propose a set of candidates equal in size to the number of open seats, and those would be the new Project Directors if the proposal is consented to.

I don't know if that distinction makes sense how I've described it? Maybe it is simpler if the "type" of the thing produced at the end of each phase is the same (meaning Teams would nominate a set of Project Directors as a unit, rather than a set of people they feel would be qualified for the role).

Reply:
I feel that ambiguity doesn't pose a risk to the process. I feel like we're at the point of having a lot of redundancy in this process, so if we aggregate the nominations into a pool and recreate sets vs. starting with sets and then possibly changing them in the change round, it all feels like it's gonna produce similarly high quality outputs.

Reply:
Maybe being explicit that all candidates are pooled at the start of the Election Meeting wouldn't hurt. I agree that the ambiguity doesn't necessarily hurt anyway, but it could lead to teams feeling more constrained. For example, if we have three open seats then in the Pre-meeting Coordination phase a team might only have one candidate they can think of to nominate and feel like they can't make the nomination unless they have exactly three. I would think nominating one would be good, especially if they nominate someone who wasn't otherwise considered, and nominating four would be fine too, especially because that could help if we get constrained by affiliation limits.

```suggestion
* Share nominations. Nominations from all teams will be pooled together. In other words, teams do not have to nominate a set of directors as a whole. They may nominated any number of potential directors and the Council will select from the pool a number of directors equal to the number of open seats.
```